src/vos/tests/evt_ctl.sh

```diff
@@ -167,6 +158,11 @@ echo "$cmd"
 
 $cmd -t
 
-result=$?
-echo Test returned $result
-exit $result
+if [[ -z ${CMOCKA_MESSAGE_OUTPUT+x} || -z ${CMOCKA_XML_FILE+x} ]]; then
+	result=$?
+	echo "Test returned $result"
+	exit $result
```

Comment:
Should quote the variable usage in the exit also:
```suggestion
	exit "$result"
```

Reply:
Updated.